spec/request_handler/multipart_parser_spec.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'request_handler/multipart_parser'
+describe RequestHandler::MultipartsParser do
```

Comment:
can you add a test which uses a full MockRequest like described [here](http://stackoverflow.com/a/25509351)